apps/parsley/src/main.tsx

```diff
@@ -12,16 +12,21 @@ import {
   isDevelopmentBuild,
 } from "utils/environmentVariables";
 import App from "./App";
-import routes from "./constants/routes";
+import routes, { slugs } from "./constants/routes";
 
+const routeConfig = {
+  ...routes,
+  testLogs: `${routes.testLogs}/:${slugs.groupID}?`,
```

Comment:
Why do we want to overwrite this value for Honeycomb? Maybe worth leaving a comment.

Reply:
It helps with making sure we can identify URLs with the groupID slugs. Otherwise, the ReactRouterSpanProcessor just thinks its a different URL entirely.